hoot-core/src/main/cpp/hoot/core/ops/RemoveRoundabouts.cpp

```diff
@@ -92,13 +96,19 @@ void RemoveRoundabouts::removeRoundabouts(std::vector<RoundaboutPtr> &removed)
 
       // If no sibling, do the mangle
       if (!foundSibling[i])
+      {
         removed[i]->handleCrossingWays(_pMap);
+        removed[i]->overrideRoundabout();
```

Comment:
No sibling (or matching roundabout) was found so the override flag must be set to get the roundabout back in the output data.